### DISCUSSION TITLE
Get rid of shaders warning

### DISCUSSION THREAD

**Discussion by Ta1kativ** [iPortalTeam/ImmersivePortalsMod#1555] [Q&A]
Every time join a world on Minecraft, I get this message in red text in the chat:

> [Immersive Portals] The current shaderpack is incapatable with Immersive Portals. Portal rendering may be very broken. If you want better portal rendering, it's recommended that you try another shaderpack.

I have used this shaderpack many times, and it works just fine with IP, and I'm tired of seeing this large red message every time I get on and play. I've looked through the settings but don't see a button to disable it. Is there a way to do so?

**Comment by qouteall** [comment]
Which shaderpack are you using?

**Reply by Ta1kativ** [reply]
I'm using complimentary shaders. It works just fine with IP on my machine

**Reply by qouteall** [reply]
These are issue reports related with complimentary shader
https://github.com/iPortalTeam/ImmersivePortalsMod/issues/1381
https://github.com/iPortalTeam/ImmersivePortalsMod/issues/1527

**Reply by qouteall** [reply]
Is compllimentary and complimentary reimagined two different shaders?

**Reply by Nick1st** [reply]
They are the same shader, only different settings, iirc.

**Reply by SkyExploreWasTaken** [reply]
> These are issue reports related with complimentary shader #1381 #1527

The only real bug is shadows flickering a lot, but it's not very game-changing
